examples/celllistmap.jl

```diff
@@ -213,9 +213,11 @@ model = initialize_bouncing(number_of_particles=1000)
 abmvideo(
     "celllistmap.mp4", model;
     framerate=20, frames=200, dt=5,
-    title="Softly bouncing particles with CellListMap.jl",
+    title="Softly Bouncing Particles – CellListMap.jl",
```

Comment:
```suggestion
    title="Softly Bouncing Particles with CellListMap.jl",
```

seems better, because CellListMap.jl is a different library, and in all other examples use "with"

Reply:
e.g. in https://juliadynamics.github.io/Agents.jl/stable/examples/schoolyard/#Social-networks-with-Graphs.jl